### DISCUSSION TITLE
question on running all tests!

### DISCUSSION THREAD

**Discussion by sozoalvin** [nvim-neotest/neotest#204] [General]
hi there! is there a command that we can run to run all the tests that are found in the local code?

**Comment by marilari88** [comment]
Hi @sozoalvin , you can run:
`:lua require("neotest").run.run(vim.fn.getcwd())`

You can also create custom commands in your config files:

```lua
vim.cmd([[
    command! NeotestSummary lua require("neotest").summary.toggle()
    command! NeotestFile lua require("neotest").run.run(vim.fn.expand("%"))
    command! Neotest lua require("neotest").run.run(vim.fn.getcwd())
    command! NeotestNearest lua require("neotest").run.run()
    command! NeotestDebug lua require("neotest").run.run({ strategy = "dap" })
    command! NeotestAttach lua require("neotest").run.attach()
]])
```

**Reply by sozoalvin** [reply]
thank you - i'll be sure to try it out

**Reply by rcarriga** [reply]
Just to add if you're not always in your root directory, you can pass `suite = true` to the `run` function, see `:h neotest.run.run()`
